packages/schematics/angular/migrations/update-8/index.ts

```diff
@@ -0,0 +1,103 @@
+/**
```

Comment:
why is this the `index` file? aren't there a bunch of migrations for 8? I'd expect this to be update-codelyzer or something

Reply:
I have a follow up PR for polyfills migration. I'll update the script there.